guides/micronaut-k8s-oci/oci-delete-repository.adoc

```diff
@@ -1,7 +1,7 @@
 Run the next command to delete `micronaut-k8s/{0}` artifacts container repository:
 
-[source,bash]
+[source,bash,subs="attributes+"]
 ----
-oci artifacts container repository delete --repository-id ${0:U}_REPOSITORY --force
+oci artifacts container repository delete --repository-id ${0_U}_REPOSITORY --force
```

Comment:
why replacing : with _

Reply:
it's again needed because otherwise asciidoc would have problems parsing it (it conflicts with attribute. declaration, which is done like :arg: user). At the moment it doesn't have any effect.

Reply:
> At the moment it doesn't have any effect.

What do you mean it has no effect. With this change it will render : 

```
oci artifacts container repository delete --repository-id ${0_U}_REPOSITORY --force
```

instead of this as it was in the guide: 
```
oci artifacts container repository delete --repository-id ${0:U}_REPOSITORY --force
```

Reply:
No. It was never rendered as ${0:U}. Asciidoc replaces {0:U} with the corresponding argument to uppercase, before the document is passed to asciidoctor. Indeed if you open the final HTML (which is what we care about) it is rendered as `oci artifacts container repository delete --repository-id $USERS_REPOSITORY --force`